app/views/DistributionView.scala.html

```diff
@@ -29,7 +29,17 @@
     @formHelper(action = routes.DistributionController.onSubmit(mode), 'autoComplete -> "off") {
 
         @if(form.errors.nonEmpty) {
-            @govukErrorSummary(ErrorSummaryViewModel(form,  errorLinkOverrides = Map("value" -> "value_0")))
+            @govukErrorSummary(
+                ErrorSummary(
+                    errorList = Seq(
+                        ErrorLink(
+                            href = Some("#value_0"),
+                            content = Text(messages("distribution.error.required"))
+                        )
+                    ),
+                    title = Text(messages("distribution.errorsummary")),
+                )
+            )
```

Comment:
Not sure why this is needed. Can we have a chat when you are available?

Reply:
Updated and simplified this